src/kvstore/raftex/RaftPart.cpp

```diff
@@ -1091,7 +1175,7 @@ bool RaftPart::leaderElection() {
     using namespace folly;  // NOLINT since the fancy overload of | operator
 
     cpp2::AskForVoteRequest voteReq;
-    decltype(hosts_) hosts;
+    std::vector<std::shared_ptr<Host>> hosts;
     if (!prepareElectionRequest(voteReq, hosts)) {
         // Suppose we have three replicas A(leader), B, C, after A crashed,
         // B, C will begin the election. B win, and send hb, C has gap with B
```

Comment:
the sentence 'Meanwhile C begin the election' .It means there is a situation where C restarts the election after B win.
In fact, there is one condition for C to resume the election after B win,
that is, the heartbeat between B and C has timedout due to pressure and other reasons?